ansible/roles/sonicv2/tasks/sonic-cavm.yml

```diff
@@ -16,9 +16,18 @@
     src: ssw_extra/{{ sonic_hwsku }}/port_config_{{ sonic_portsku }}.ini
     dest: /etc/ssw/{{ sonic_hwsku }}/port_config.ini
 
+# Setup profile for syncd
+- name: Setup profile configuration file
+  become: yes
+  template: src=etc/ssw/{{ sonic_hwsku }}/profile.ini.j2
+            dest=/etc/ssw/{{ sonic_hwsku }}/profile.ini
+            owner=root
+            group=root
+            mode=0644
+
```

Comment:
please add profile https://github.com/Azure/sonic-sairedis/tree/master/etc/syncd.d

the script to select profile based on the platform is https://github.com/Azure/sonic-sairedis/blob/master/debian/syncd.init